ruby/ext/google/protobuf_c/message.c

```diff
@@ -44,6 +44,10 @@ void Message_mark(void* _self) {
 }
 
 void Message_free(void* self) {
+  void* unknown = ((MessageHeader *)self)->unknown_fields;
+  if (unknown != NULL) {
+    free(unknown);
```

Comment:
stringsink_uninit? do we have a heap checker? this looks like a leak.

Reply:
Forgot to push latest change.
Currently, we don't have heap checker.